### PR TITLE
Add segmentation-fold

### PR DESCRIPTION
**Pull request by shiltemann**
Please sign off by admins only
------------------------------

* [x] Has no duplicate (otherwise merge required)
* [ ] Indentation is correct
* [x] Tests
  - [x] Parameters are reasonably covered
  - [x] Test files are appropriate
* [x] `<command/>`: file names are escaped
* [x] Tool requires version/build increase

### CODE REVIEW

File: tools/segmentation_fold/utils_fix-fasta-headers.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    <outputs>
+        <data format="fasta"
+              name="fasta_output"
+              label="${tool.name} on ${str($fasta_input.hid) + ': ' + $fasta_input.name}" />
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="fasta_input" value="test_22.fa" format="fasta" />
```
Comment (shiltemann) [inline tools/segmentation_fold/utils_fix-fasta-headers.xml]: s/format/ftype/

File: tools/segmentation_fold/utils_fix-fasta-headers.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            
+            <output name="fasta_output" file="test_22.fixed.fa" />
+        </test>
+        <test>
+            <param name="fasta_input" value="test_23.fa" format="fasta" />
```
Comment (shiltemann) [inline tools/segmentation_fold/utils_fix-fasta-headers.xml]: s/format/ftype/

File: tools/segmentation_fold/utils_find-boxes.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    <outputs>
+        <data format="bed"
+              name="bed_output"
+              label="${tool.name} on ${str($fasta_input.hid) + ': ' + $fasta_input.name}" />
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="fasta_input" value="FindBoxes.genome.fa" format="fasta" />
```
Comment (shiltemann) [inline tools/segmentation_fold/utils_find-boxes.xml]: s/format/ftype/

File: tools/segmentation_fold/utils_estimate-energy.xml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                    argument="-x"
+                    label="Custom 'segments.xml'-syntaxed file" />
+            </when>
+        </conditional>
+        
+        <param name="precision"
+               type="float"
+               value="0.05"
+               min="0"
+               argument="--precision"
```
Comment (shiltemann) [inline tools/segmentation_fold/utils_estimate-energy.xml:56]: alsje `argument` gebruikt hoeft `name` in princiepe niet meer

File: tools/segmentation_fold/utils_estimate-energy.xml [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                $output_list
+    ]]></command>
+
+    <inputs>
+        <conditional name="parameters">
+            <param name="use_custom_xml"
+                type="boolean"
+                truevalue="true"
+                falsevalue="false"
+                selected="false"
```
Comment (shiltemann) [inline tools/segmentation_fold/utils_estimate-energy.xml]: s/selected/checked/

File: packages/package_segmentation_fold_1_0_0/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+description: RNA-Folding including predefined segments including K-turns
+long_description: |
+  Segmentation-fold is a bioinformatics application that predicts RNA 2D-structure with an extended version of the Zuker algorithm. This modification contains a new "structure element" named a segment and is capable of folding a pre-defined substructure with multiple canonical or non-canonical pairings.
+  
+  This allows folding of more complex structures like the K-turns, which are also part of the implemented free energy tables. These thermodynamic parameters (free Gibbs energy levels) have been estimated using a computational approach and therefore lack accuracy.
+
+name: package_segmentation_fold_1_0_0
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/segmentation-fold
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/segmentation_fold_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_segmentation_fold_1_0_0/.shed.yml]: update repo url and give admin access

**Comment by yhoogstrate** [comment]
segmentation-fold needs a subprogram to obtain the path to it's default config file, and this needs to be used

**Pull request merged**